shell/golangci-lint.sh

```diff
@@ -24,4 +24,10 @@ if grep "^\d" <<<"$version"; then
   version="v$version"
 fi
 
+# Use individual directories for golangci-lint cache as opposed to a mono-directory.
+# This helps with the "too many open files" error.
+mkdir -p "$HOME/.outreach/.cache/.golangci-lint" >/dev/null 2>&1
+GOLANGCI_LINT_CACHE="$HOME/.outreach/.cache/.golangci-lint/$(get_app_name)"
+export GOLANGCI_LINT_CACHE
```

Comment:
nit: can do in the same line

```suggestion
export GOLANGCI_LINT_CACHE="$HOME/.outreach/.cache/.golangci-lint/$(get_app_name)"
```